### PR TITLE
format flutter tables

### LINKED ISSUE #219: Do not hard-code settlement amounts
With #207 we introduced the ability to actually settle CFDs (removing the custom output from the Lightning channel). But the values used are hard-coded in the frontend.

### Rough design
1. User navigates to CFD order details page (data loaded from storage, see #201).
2. User presses the `settle` button.
3. We use some of the persisted values together with the current price to call a `calculate_payouts_function` which acts as the payout curve.
4. The function returns `maker_amount` and `taker_amount` which can replace the hard-coded values.

https://github.com/itchysats/10101/blob/a91a50c3df483a95fa61e265d4671bb64286622f/lib/cfd_trading/cfd_order_detail.dart#L124

### PR DESCRIPTION
**Pull request by bonomat**
This was a bit of a failure, this PR is now the same as https://github.com/itchysats/10101/pull/252

### CODE REVIEW

File: lib/cfd_trading/cfd_order_detail.dart [hunk 1]
```diff
@@ -6,14 +6,15 @@ import 'package:flutter_rust_bridge/flutter_rust_bridge.dart';
...
 import 'package:ten_ten_one/cfd_trading/cfd_trading.dart';
-import 'package:ten_ten_one/models/amount.model.dart';
 import 'package:ten_ten_one/cfd_trading/cfd_trading_change_notifier.dart';
 import 'package:ten_ten_one/models/order.dart';
 import 'package:ten_ten_one/utilities/tto_table.dart';
 import 'package:go_router/go_router.dart';
 
 import 'package:ten_ten_one/ffi.io.dart' if (dart.library.html) 'ffi.web.dart';
 
+import 'cfd_offer_change_notifier.dart';
```
Comment (holzeis) [inline lib/cfd_trading/cfd_order_detail.dart:16]: 🔧 Please use package imports

File: lib/cfd_trading/cfd_order_detail.dart [hunk 2]
```diff
@@ -33,16 +34,33 @@ class _CfdOrderDetailState extends State<CfdOrderDetail> {
...
+    final leverage = order.leverage;
+
+    var takerAmount = 0;
+    var makerAmount = 0;
+
+    if (offer != null) {
+      final marginAsSats = order.margin.asSats;
+      // TODO: this should be stored in the order
+      final totalValueAsSats = marginAsSats + marginAsSats * leverage;
+      final takerPnlAsSats = order.pl.asSats;
```
Comment (holzeis) [inline lib/cfd_trading/cfd_order_detail.dart:59]: 🤔  Shouldn't that pl be calculated? I think this is still mocked data isn't it?

Comment (bonomat) [reply lib/cfd_trading/cfd_order_detail.dart:59]: I think so, I was hoping this is done as part of #201

File: lib/cfd_trading/cfd_order_detail.dart (hunk 1 above)
Comment (bonomat) [reply lib/cfd_trading/cfd_order_detail.dart]: fixed and force-pushed

File: lib/cfd_trading/cfd_order_detail.dart (hunk 2 above)
Comment (holzeis) [reply lib/cfd_trading/cfd_order_detail.dart:58]: fair enough, I am going to add it then 😝

Comment (holzeis) [reply lib/cfd_trading/cfd_order_detail.dart:58]: https://github.com/itchysats/10101/pull/255

**Review by luckysori — COMMENTED** [review]
The diff seems a bit weird, is it including stuff that shouldn't be here?

**Comment by bonomat** [comment]
> The diff seems a bit weird, is it including stuff that shouldn't be here?

Lol, I force pushed the wrong branch onto this 🙈

**Comment by bonomat** [comment]
> > The diff seems a bit weird, is it including stuff that shouldn't be here?
> 
> Lol, I force pushed the wrong branch onto this 🙈

force pushed the correct one again.

**Comment by bonomat** [comment]
> > > The diff seems a bit weird, is it including stuff that shouldn't be here?
> > 
> > 
> > Lol, I force pushed the wrong branch onto this 🙈
> 
> force pushed the correct one again.

what a gigantic failure... I on purpose merged https://github.com/itchysats/10101/pull/252 into this one because it was built up on...

**Comment by bonomat** [comment]
converting to draft until I fix the commits...

**Comment by bonomat** [comment]
PR is ready, please only focus on [`0d9be2f` (#248)](https://github.com/itchysats/10101/pull/248/commits/0d9be2f33e202f115acb3e5b043e56d538409e23), the other 3 commits have been reviewed already in https://github.com/itchysats/10101/pull/252

**Comment by bonomat** [comment]
bors r+

No reviews needed as this was already reviewed in https://github.com/itchysats/10101/pull/252

**Pull request merged**